src/libertem/udf/masks.py

```diff
@@ -144,10 +144,7 @@ def process_tile(self, tile):
         ''
         flat_data = tile.reshape((tile.shape[0], -1))
         if self.task_data.use_torch:
-            try:
```

Comment:
The purpose of `try ... except` is to work without installed torch since that is an optional dependency.

Reply:
Is that still required if use_torch is set, or is use_torch only set if torch can be imported?

Reply:
True, but when we enter that branch, `use_torch` is `True`, which is only the case if the import in `get_task_data` worked